.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
```

Comment:
Can yo add another one for stack overflow? https://stackoverflow.com/questions/tagged/vowpalwabbit

Reply:
sure